sdk/iot/Azure.Iot.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -144,6 +144,41 @@ public partial class BulkRegistryOperationResponse
         public bool? IsSuccessful { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<Azure.Iot.Hub.Service.Models.DeviceRegistryOperationWarning> Warnings { get { throw null; } }
     }
+    public partial class CloudToDeviceMessage
```

Comment:
Is this a generated file?

Reply:
No, this is the customized sub-client that we are defining in the SDK. This is similar to `DevicesClient` and `ModulesClient` that we currently have.

Reply:
We should add xml comments to the file.

Reply:
That'll be added when we actually implement this API surface.